epispot/__init__.py

```diff
@@ -1,28 +1,173 @@
+                                 >>> pip install matplotlib  # pip
+                                 >>> conda install matplotlib  # anaconda''')
+
+
 # imports
 import warnings
 import random
+
+
+# dependencies
+_dependency_check()  # check for uninstalled dependencies
 import numpy as np
 from matplotlib import pyplot as plt
 from matplotlib import colors
 
+
 # local
 from . import comps
 from . import models
 from . import pre
 from . import plots
 from . import fitters
 
+
+def _sanity_check():
+    """Pre-defined sanity check to check for installation errors"""
+
+    if not version or not source:  # pragma: no cover
+        message = \
+        '''
+        Version information and/or source for your epispot installation could not be found.
+        This likely means
+         (1) this version of epispot is deprecated or dated
+         (2) your installation of epispot is unstable
+        Try reinstalling epispot
+         (1) From PyPI:
+             $ pip install epispot
+         (2) From Anaconda:
+             $ conda config --add channels conda-forge
+             $ conda install -c conda-forge epispot
+         (3) From the GitHub Source:
+             $ git clone https://github.com/epispot/epispot
+             $ cd epispot
+             $ python install setup-nightly.py
+        If this fails or has failed before, then revert to epispot's last stable version.
+        Again, you can do this via
+         (1) PyPI:
+             (a) Go to https://pypi.org/project/epispot
+             (b) Scroll to the last MAJOR release in the form `a.0.0`
+             (c) Install via `$ pip install epispot==a.0.0` -- replace `a` with version #
+         (2) GitHub Source:
+             (a) Go to https://github.com/epispot/epispot
+             (b) Click the 'branches' tab
+             (c) Select the last branch with name `vA.0.0` for some version number A
+             (d) Install as `.zip`
+             (e) Unzip
+             (f) Run `$ python install setup.py`
+        '''
+        raise DeprecationWarning(message)
+
+
 # version info
-__version__ = "v2.1.1.7"  # version (v#.#.#)
-"""epispot's version info"""
-__stable__ = False
-"""stability of the build: True --> stable release (master), False --> unstable release (nightly)"""
-__repo_data_url__ = 'https://www.github.com/epispot/epispot'  # source repo
+version = '2.1.1.7'
+"""
+Epispot's version info (updated every nightly release)
+Get with 
+    >>> print(epispot.version)
+"""
+
+stable = False
+"""
+Build stability
+- True --> main package (stable)
+- False --> nightly package (possibly unstable)
+"""
+
+# metadata
+source = 'https://www.github.com/epispot/epispot'
 """URL to GitHub source"""
+docs = 'https://epispot.github.io/epispot'
+"""Project documentation (not version-specific)"""
+issues = 'https://www.github.com/epispot/epispot/issues'
+"""Submit new issues or bugs here"""
+
+
+_sanity_check()  # complete installation
```

Comment:
You shouldn't have the sanity check after running it.
```suggestion
_sanity_check()  # complete installation
del _sanity_check()
```